docs/root/intro/arch_overview/other_protocols/redis.rst

```diff
@@ -9,7 +9,7 @@ over consistency. This is the key point when comparing Envoy to `Redis Cluster
 <https://redis.io/topics/cluster-spec>`_. Envoy is designed as a best-effort cache,
 meaning that it will not try to reconcile inconsistent data or keep a globally consistent
 view of cluster membership. It also supports routing commands from different workload to
```

Comment:
@agrawalnishant thanks for this - i think `workload` here should also be plural - ie `workloads` could you fix this also please